iree/hal/vulkan/api.h

```diff
@@ -30,19 +30,13 @@ extern "C" {
 // Types and Enums
 //===----------------------------------------------------------------------===//
 
-// Bitfield that defines Vulkan extensibility options to enable.
+// Bitfield that defines Vulkan extensibility options to enable, if available.
 typedef enum {
-  // Enables standard Vulkan validation layers.
-  IREE_HAL_VULKAN_ENABLE_VALIDATION_LAYERS = 0,
-
   // Enables VK_EXT_debug_utils, records markers, and logs errors.
   IREE_HAL_VULKAN_ENABLE_DEBUG_UTILS = 1 << 0,
 
-  // Enables VK_EXT_debug_report and logs errors.
-  IREE_HAL_VULKAN_ENABLE_DEBUG_REPORT = 1 << 1,
-
-  // Enables use of vkCmdPushDescriptorSetKHR, if available.
-  IREE_HAL_VULKAN_ENABLE_PUSH_DESCRIPTORS = 1 << 2,
+  // Enables use of vkCmdPushDescriptorSetKHR.
+  IREE_HAL_VULKAN_ENABLE_PUSH_DESCRIPTORS = 1 << 1,
 } iree_hal_vulkan_extensibility_options_t;
```

Comment:
rename to iree_hal_vulkan_driver_options_t, as it's options being set (not the ability to set options)

Reply:
I'm not quite following this comment.

This bitfield is used by both `iree_hal_vulkan_driver_create` (with the options that you want to be enabled) and `iree_hal_vulkan_driver_create_using_instance` (with the options that you have already enabled, hopefully by asking with `iree_hal_vulkan_get_extensions`).

There is also already a `iree_hal_vulkan_driver_options_t`. Is there a different name that you want here?